main.typ

```diff
@@ -38,31 +38,51 @@
 
-Despite these alternative management models, numerous expansive and widely utilized open-source projects persist under the ultimate guidance of a Benevolent Dictator. A prime illustration of this is the Linux kernel, where Linus Torvalds assumes the role of the Benevolent Dictator For Life. However, beneath his purview exists a hierarchical structure comprising maintainers tasked with overseeing distinct facets of the kernel. These maintainers assume the responsibility of scrutinizing and endorsing the quality of submitted code as it ascends the chain of command, ultimately reaching Linus Torvalds. #cite(<user229044_2010>) #cite(<How-Linux-is-Built>)
+Despite these alternative management models, numerous expansive and widely utilized open-source projects persist under the ultimate guidance of a Benevolent Dictator. A prime illustration of this is the Linux kernel, where Linus Torvalds assumes the role of the Benevolent Dictator For Life. However, beneath his purview exists a hierarchical structure comprising maintainers tasked with overseeing distinct facets of the kernel. These maintainers assume the responsibility of scrutinizing and endorsing the quality of submitted code as it ascends the chain of command, ultimately reaching Linus Torvalds @user229044_2010 @How-Linux-is-Built.
 
 == Comparing Agile and Open Source Management Styles
+Agile and Open Source management styles are quite similar in a few different aspects. Regarding the process of developing software, while agile methods have predefined processes and sequences, numerous open source projects also have structured processes @Eckstein_2004. Agile development requires close collaboration within the team (in methods of pair programming, review, etc), whereas open source teams are scattered in different parts of the world, working asyncronously. 
+// discuss customer/stakeholder interactions, early delievery and feedback, team structure
 
+== The Impact of Agile Methods on Open Source Development
+As Agile Methodologies integrate into mainstream development due to the positive effects on code quality and quick response time,aspects of agile are emerging into open source development. A trend in open source projects appear to be the adoption of Testing and Continuous Integration. The implementation of automated tests establishes trust among the developers involved with the open source project because passing tests demonstrates that their code works, and that the code does not have to be rewritten by other developers @Turnu_2006.
```

Comment:
Honestly, yeah this is pretty good insight. Early open source projects didn't really do anything about CI and that's something I take for granted. The earliest commit from busybox I could find referencing testing was in 1999 https://git.busybox.net/busybox/commit/?id=a89910829c73c5ed4f30eecb181d7b3087ca4f74, which is significantly after most of the tools in busybox were introduced by bell labs.